usr.bin/du/du.c

```diff
@@ -453,9 +466,10 @@ linkchk(FTSENT *p)
 		/* Pull a node from the free list if we can. */
 		le = free_list;
 		free_list = le->next;
-	} else
+	} else {
 		/* Malloc one if we have to. */
 		le = malloc(sizeof(struct links_entry));
+	}
```

Comment:
This is a good change, but it's gratuitous since you're not changing adjacent lines.